resources/rpm/spec.erb

```diff
@@ -19,7 +19,9 @@ Name: <%= name %>
 Version: <%= version %>
 Release: <%= iteration %><%= dist_tag %>
 Summary:  <%= description.split("\n").first.empty? ? "_" : description.split("\n").first %>
+-<% if platform != 'nexus' -%>
 BuildArch: <%= architecture %>
+-<% end -%>
```

Comment:
This feels related to https://github.com/chef/omnibus/pull/542. We might not need `BuildArch` in the spec file.